static/docs/command-reference/checkout.md

```diff
@@ -94,6 +88,12 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - recreates (overwrites) file links or copies, from cache to
```

Comment:
`recreates (overwrites) file links or copies` - too complicated. If we'd like to be 100% precise (I'm not sure it's needed to such extent), let's write two sentences.

Reply:
OK. I rewrote it in 83757462 to be both simpler and fully correct (I think):

```
- `--relink` - recreates **all <abbr>outputs</abbr>** referenced in current
  DVC-files (regardless of whether the checksums match a DVC-file). This means
  overwriting the file links or copies from cache to workspace. This ensures the
  link types of all the data files in the workspace are consistent with the
  project's [`cache.type`](/doc/command-reference/config#cache).
```

Reply:
> I would even prefer to just for the second sentence go first. Then some explanation if needed.

Good idea, just changed to:

```
- `--relink` - ensures the link types of all the data files in the workspace are
  consistent with the project's
  [`cache.type`](/doc/command-reference/config#cache). This is achieved by
  recreating **all <abbr>outputs</abbr>** referenced in current DVC-files
  (regardless of whether the checksums match a DVC-file). This means overwriting
  the file links or copies from cache to workspace.
```